portal-bridge/src/bridge/era1.rs

```diff
@@ -161,9 +162,13 @@ impl Era1Bridge {
             .recv_bytes()
             .await
             .unwrap_or_else(|_| panic!("unable to read era1 file at path: {era1_path:?}"));
-        let era1 = Era1::deserialize(&raw_era1).expect("to be able to deserialize era1 file");
-        let epoch_index = era1.block_tuples[0].header.header.number / EPOCH_SIZE as u64;
-        info!("Era1 file read successfully, gossiping block tuples for epoch: {epoch_index}");
+        let epoch_index = match get_epoch_from_era1_path(&era1_path) {
+            Ok(epoch) => epoch,
+            Err(e) => {
+                error!("Failed to get epoch from era1 file: {e}");
+                return;
```

Comment:
```suggestion
                panic!("Failed to get epoch from era1 file: {e}");
```

Wouldn't this make more sense as a panic! as this ends the program either way?